README.md

```diff
@@ -23,36 +23,61 @@ Installation
 ember install ember-cli-content-security-policy
 ```
 
-## Configuration
+Configuration
+------------------------------------------------------------------------------
 
 This addon is configured via `config/content-security-policy.js` file.
 
-- `delivery: string[]`
-  CSP is delivered via HTTP Header if delivery includes `"header"` and via meta element if it includes `"meta"`.
-  Defaults to `["header"]`.
-- `enabled: boolean`
-  Controls if addon is enabled at all.
-  Defaults to `true`.
-- `policy: object`
-  A hash of options representing a Content Security Policy.
-  Defaults to:
-  ```js
-  {
-    'default-src':  ["'none'"],
-    'script-src':   ["'self'"],
-    'font-src':     ["'self'"],
-    'connect-src':  ["'self'"],
-    'img-src':      ["'self'"],
-    'style-src':    ["'self'"],
-    'media-src':    ["'self'"],
-  }
-  ```
-  To clear a directive from the default policy, set it to `null`.
```

Comment:
Since the configuration is not merged anymore, there isn't any need to set a directive to `null`. Missed to delete that sentence in #101.